src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -232,6 +233,11 @@ static void setNotification(Subscription* subP, const orion::BSONObj& r, const s
       subP->notification.lastSuccess     = cSubP->lastSuccess;
       subP->notification.lastSuccessCode = cSubP->lastSuccessCode;
     }
+
+    if (cSubP->covered > subP->notification.covered)
```

Comment:
This doesn't make sense... cSubP->covered is not numerical, but a bool.

The treatment should be pretty similar to boolean cSubP->onlyChanged